Integrations/FeedTAXII/FeedTAXII_description.md

```diff
@@ -9,3 +9,41 @@ To connect the TAXII Feed integration to a TAXII server you'll need to configure
 subscription a *Subscription ID* value. From then on, both the Consumer and the Server refer to this subscription in messages using this Subscription ID value.
 * **Poll Service** (Optional) - Used by a TAXII Client to request information from a TAXII Server. If not provided, will be fetched from the discovery service.
 * **API Key + API Header** (Optional) - API key value and the header name to be presented to the TAXII server for authentication.
+
+
+## Step by step configuration
```

Comment:
in general, i think it would be great if you mention for each parameter why it should be set to the suggested value, like blank or `80`